core/src/error_utils.rs

```diff
@@ -0,0 +1,24 @@
+use core::fmt::{self, Debug, Display};
+
+#[derive(Clone, PartialEq, Eq)]
+pub struct DisplayError<T>(pub T);
```

Comment:
Offhand I can't quite see what the reason for this is?

Reply:
> snafu uses its own custom snafu::Error type in no-std contexts, therefore we needed to define a wrapper type that implements snafu::AsErrorSource for external error types and this led to a bit of boilerplate.snafu uses its own custom snafu::Error type in no-std contexts, therefore we needed to define a wrapper type that implements snafu::AsErrorSource for external error types and this led to a bit of boilerplate.

Most likely :point_up:

Reply:
Ahh, I should have paid more attention reading :)